docs/source/api/python_library.rst

```diff
@@ -1,6 +1,105 @@
 Python library
 ==============
 
-This section only contains code reference (autogenerated docs are possible with a description of arguments and return values)
 
-.. Attention:: Contents are missing for now.
\ No newline at end of file
+About Nexus
+---------------
+https://nexus.soramitsu.co.jp/ - our new PyPI, Maven, artifacts server with precompilled librares.
+You can download binaries without compilation on the client side. It helps to save you time and disk space for installing dependencies like libboost, libz, etc.
+
+Now our packages are distributed only for ``Windows x86_64`` and ``Linux x86_64`` with ``Python 2.7.x`` and ``Python 3.5.x``. In future it will be available on ``Mac OS``.
+
+https://nexus.soramitsu.co.jp/repository/pypi-develop/ - developer wheels (builds from ``develop`` branch)
+https://nexus.soramitsu.co.jp/repository/pypi-release/ - release wheels (builds from ``master`` branch)
+https://nexus.soramitsu.co.jp/repository/pypi-develop-nightly/ - dev nightly wheels (daily builds from ``develop`` branch)
+
+
+Pulling from PyPI
+---------------
+
+You can download our PyPI package from our Nexus server (instead of pypi.org). To do this, please specify ``-i`` parameter:
+
+From develop
+
+.. code-block:: shell
+
+    pip install -i https://nexus.soramitsu.co.jp/repository/pypi-develop/ iroha 
```

Comment:
Does not work on a clean Ubuntu 18.04 installation (neither python3 nor python2). Please verify